cds/modules/records/serializers/schemas/datacite.py

```diff
@@ -101,23 +102,27 @@ def get_creators(self, obj):
         """Get creators."""
         items = []
         for item in obj["metadata"].get("contributors", []):
-            items.append(
-                {
-                    "creatorName": item.get("name", ""),
-                }
+            if item.get("role", "") != "Research Group":
+                items.append(
+                    {
+                        "creatorName": item.get("name", ""),
+                    }
             )
         return items
 
-    #  def get_contributors(self, obj):
-    #      """Get contributors."""
-    #      items = []
-    #      for item in obj['metadata'].get('contributors', []):
-    #          items.append({
-    #              'contributorType': item.get('role', ''),
-    #              'contributorName': item.get('name', ''),
-    #              # FIXME nameIdentifier and nameIdentifierScheme, ... ?
-    #          })
-    #      return items
+    def get_contributors(self, obj):
+        """Get contributors."""
+        items = []
+        for item in obj['metadata'].get('contributors', []):
+            if item.get("role", "") == "Research Group":
+                items.append(
+                    {
+                    'contributorType': item.get('role', '').replace(" ", ""),
```

Comment:
Here we need to check from the available roles which exist in datacite and if they don't we should assign `contributorType: Other`